filecoin-proofs/src/constants.rs

```diff
@@ -47,24 +47,47 @@ pub const PUBLISHED_SECTOR_SIZES: [u64; 10] = [
     SECTOR_SIZE_64_GIB,
 ];
 
+pub struct PorepMinimumChallenges(RwLock<HashMap<u64, usize>>);
+impl PorepMinimumChallenges {
+    fn new() -> Self {
+        Self(RwLock::new(
+            [
+                (SECTOR_SIZE_2_KIB, 2),
+                (SECTOR_SIZE_4_KIB, 2),
+                (SECTOR_SIZE_16_KIB, 2),
+                (SECTOR_SIZE_32_KIB, 2),
+                (SECTOR_SIZE_8_MIB, 2),
+                (SECTOR_SIZE_16_MIB, 2),
+                (SECTOR_SIZE_512_MIB, 2),
+                (SECTOR_SIZE_1_GIB, 2),
+                (SECTOR_SIZE_32_GIB, 176),
+                (SECTOR_SIZE_64_GIB, 176),
+            ]
+            .iter()
+            .copied()
+            .collect(),
+        ))
+    }
+
+    pub fn get_mut(&self) -> RwLockWriteGuard<'_, HashMap<u64, usize>> {
+        self.0.write().expect("POREP_MINIMUM_CHALLENGES poisoned")
+    }
+
+    pub fn porep_partitions(&self, sector_size: u64) -> usize {
```

Comment:
To me, the name of this function implies that we are deriving the partition count from the number of porep challenges. I think that a better name for this function would be something like `from_sector_size` or `for_sector_size` as we are retrieving the minimum challenge count for a given sector size.

Reply:
I agree with you, will make it more readable. Changes done.

Reply:
Awesome, thank you!